troubleshooting/known_issues/kafka_chart_upgrade_failure.md

```diff
@@ -0,0 +1,151 @@
+# cray-kafka-operator chart failing to upgrade
+
+## [Issue description](#issue-description)
+
+If a cluster which was initially installed with CSM 1.3 or earlier, and is now getting upgraded to CSM 1.6.0 or CSM 1.6.1 from CSM 1.5.x, the upgrade of the cray-kafka-operator helm chart would fail as part of the CSM services upgrade process.
```

Comment:
```suggestion
If a cluster which was initially installed with CSM 1.3 or earlier, and is now getting upgraded to CSM 1.6.x from CSM 1.5.x, the upgrade of the cray-kafka-operator helm chart would fail as part of the CSM services upgrade process.
```